stresstest-aws/bootstrap.tf

```diff
@@ -64,18 +78,37 @@ resource "local_file" "genesis-file" {
       "eip155Block": 0,
       "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
       "eip158Block": 0,
+%{if var.native_geth == false~}
       "isQuorum": true,
+%{endif~}
 %{if var.consensus == "ibft"~}
       "istanbul": {
         "epoch": 30000,
         "policy": 0,
         "ceil2Nby3Block": 0
       },
+%{endif~}
+%{if var.consensus == "clique"~}
+      "clique": {
+        "epoch": 30000,
+        "period": ${var.blockPeriod}
+      },
 %{endif~}
       "maxCodeSize": 50
     },
     "difficulty": "${var.consensus == "ibft" ? "0x1" : "0x0"}",
-    "extraData": "${var.consensus == "ibft" ? quorum_bootstrap_istanbul_extradata.this.extradata : "0x0000000000000000000000000000000000000000000000000000000000000000"}",
+%{if var.consensus == "ibft"~}
+    "extraData": "${quorum_bootstrap_istanbul_extradata.this.extradata}",
+%{endif~}
+
+%{if var.consensus == "clique"~}
+    "extraData": "0x0000000000000000000000000000000000000000000000000000000000000000${local.clique_extra_data}0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+%{endif~}
+
+
+%{if var.consensus == "raft"~}
+    "extraData": "0x0000000000000000000000000000000000000000000000000000000000000000",
+%{endif~}
```

Comment:
as we have 3 if conditions, i would declare a local map variable for consensus extra data and retrieve them with key.
```
locals {
   consensus_extra_data = {
     "istanbul" = "${quorum_bootstrap_istanbul_extradata.this.extradata}"
     "clique" = "${local.clique_extra_data}"
   }
}
....

   "extraData": "${lookup(local.consensus_extra_data, var.consensus, "0x0000000000000000000000000000000000000000000000000000000000000000")}"
```